### PR TITLE
Add WhitestormJS project

### PR DESCRIPTION
**Pull request by jackdalton**

### CODE REVIEW

File: files/whitestormjs/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "whitestormjs",
+  "repo": "sasha240100/WhitestormJS",
+  "files": {
+    "include": ["build/whitestorm.min.js", "build/whitestorm.js"]
```
Comment (jimaek) [inline files/whitestormjs/update.json]: There is no `build` directory here https://github.com/sasha240100/WhitestormJS/tree/v0.1-beta.3
The bot will fail

Comment (jackdalton) [inline files/whitestormjs/update.json]: Good catch, I'll work on that.

**Comment by jackdalton** [comment]
Fixed, messy releases deleted. Should be good to go now.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**